docs/versioned_docs/version-2.11.0/troubleshooting/t-cstor.md

```diff
@@ -0,0 +1,513 @@
+I0802 18:35:13.868117 6 run_backup_controller.go:41] Waiting for informer caches to sync
+I0802 18:35:13.946730 6 run_pool_controller.go:45] Starting CStorPool workers
+I0802 18:35:13.946931 6 run_pool_controller.go:51] Started CStorPool workers
+I0802 18:35:13.968344 6 run_replica_controller.go:47] Starting CStorVolumeReplica workers
+I0802 18:35:13.968441 6 run_replica_controller.go:54] Started CStorVolumeReplica workers
+I0802 18:35:13.968490 6 run_restore_controller.go:46] Starting CStorRestore workers
+I0802 18:35:13.968538 6 run_restore_controller.go:53] Started CStorRestore workers
+I0802 18:35:13.968602 6 run_backup_controller.go:46] Starting CStorBackup workers
+I0802 18:35:13.968689 6 run_backup_controller.go:53] Started CStorBackup workers
+I0802 18:35:43.869876 6 handler.go:456] cStorPool pending: 48d3b2ba-b553-11e9-858e-54e1ad4a9dd4
+I0802 18:35:43.869961 6 new_pool_controller.go:160] cStorPool Modify event : cstor-sparse-p8yp, 48d3b2ba-b553-11e9-858e-54e1ad4a9dd4
+I0802 18:35:43.870552 6 event.go:221] Event(v1.ObjectReference{Kind:"CStorPool", Namespace:"", Name:"cstor-sparse-p8yp", UID:"48d3b2ba-b553-11e9-858e-54e1ad4a9dd4", APIVersion:"openebs.io/v1alpha1", ResourceVersion:"2070", FieldPath:""}): type: 'Normal' reason: 'Synced' Received Resource modify event
+I0802 18:35:44.905633 6 pool.go:93] Import command successful with true dontimport: false importattr: [import -c /tmp/pool1.cache -o cachefile=/tmp/pool1.cache cstor-48d3b2ba- b553-11e9-858e-54e1ad4a9dd4] out:
+```
+
+From the above highlighted logs, we can confirm `cstor-pool-mgmt` in new pod is communicating with `cstor-pool` in old pod as first highlighted log says `cstor pool found` then next highlighted one says pool is really `imported`.
+
+**Possible Reason:**
+
+When a cstor pool pod is deleted there are high chances that two cstor pool pods of same pool can present i.e old pool pod will be in `Terminating` state(which means not all the containers completely terminated) and new pool pod will be in `Running` state(might be few containers are in running state but not all). In this scenario `cstor-pool-mgmt` container in new pool pod is communicating with `cstor-pool` in old pool pod. This can cause CVR resource to set to `Invalid`.
+
+**Note:** This issue has observed in all OpenEBS versions up to 1.2.
+
+**Resolution:**
+
+Edit the `Phase` of cStorVolumeReplica (cvr) from `Invalid` to `Offline`. After few seconds CVR will be `Healthy` or `Degraded` state depends on rebuilding progress.
+
+### cStor volume become read only state {#cstor-volume-read-only}
+
+Application mount point running on cStor volume went into read only state.
+
+**Possible Reason:**
+
+If `cStorVolume` is `Offline` or corresponding target pod is unavailable for more than 120 seconds(iSCSI timeout) then the PV will be mounted as `read-only` filesystem. For understanding different states of cStor volume, more details can be found [here](/docs/next/kb.html#verification-of-cStor-storage-volume).
```

Comment:
Do we have this `next` route?

Reply:
This is the old URL structure. Can update these URLs to new pages? Few sections have not yet been imported but we can still update the URL according to the section. For example, this URL should be `/docs/additional_info/kb#verification-of-cStor-storage-volume`

Reply:
I see it in multiple places. Please change this in all places where `next` is used

Reply:
There are two types of internal links.
The first type of internal links page is already present in the code.
For the second type of internal links, docs are yet to migrate.

The second type of internal links will be changed once all docs will be migrated.

Reply:
Since all the other sections in the docs are being migrated soon, can we add those URLs since we know the section names already?

Reply:
Let's fix it in broken links as separate ( otherwise we would have to do guesswork).
replied it on slack threads.